### PR TITLE
Update vectorgeoprocessing.rst

### PR DESCRIPTION
**Pull request by havatv**
Update to table description (#2770).
Fixes: #3396

Goal:

Ticket(s): #

- [x] Backport to LTR documentation is required

### CODE REVIEW

**Comment by havatv** [comment]
I will merge this one as soon as travis is happy, so that we can see how this table setup works for the description of the parameters (as discussed in #2770).

**Comment by SrNetoChan** [comment]
> I will merge this one as soon as travis is happy, so that we can see how this table setup works for the description of the parameters (as discussed in #2770).

Good call!

File: source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst [hunk 1]
```diff
@@ -19,53 +19,224 @@ This algorithm is derived from the `ogr2ogr utility <https://gdal.org/ogr2ogr.ht
...
+   :stub-columns: 0
+
+   *  - Label
+      - Name
+      - Type
+      - Description
+   *  - **Dissolved**
+      - ``OUTPUT``
+      - [vector:any]
+      - The output mulitpart geometry layer (with dissolved geometries)
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst]:
```suggestion
      - The output multipart geometry layer (with dissolved geometries)
```

File: source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst [hunk 2]
```diff
@@ -19,53 +19,224 @@ This algorithm is derived from the `ogr2ogr utility <https://gdal.org/ogr2ogr.ht
...
+   :widths: 20 20 20 40
+   :stub-columns: 0
+
+   *  - Label
+      - Name
+      - Type
+      - Description
+   *  - **Dissolved**
+      - ``OUTPUT``
+      - [vector:any]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst]:
```suggestion
      - [vector: any]
```

File: source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst [hunk 3]
```diff
@@ -19,53 +19,224 @@ This algorithm is derived from the `ogr2ogr utility <https://gdal.org/ogr2ogr.ht
...
+        Default: False
+      - Compute the area and perimeter of dissolved features and include
+        them in the output layer
+   *  - **Compute min/max/sum/mean for attribute**
+      - ``COMPUTE_STATISTICS``
+      - [boolean]
+        
+        Default: False
+      - Calculate statistics (min, max, sum and mean) for the numeric
+        attribute specified, include it in the output layer
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst]:
```suggestion
        attribute specified and include them in the output layer
```
?

File: source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst [hunk 4]
```diff
@@ -19,53 +19,224 @@ This algorithm is derived from the `ogr2ogr utility <https://gdal.org/ogr2ogr.ht
...
+      - The name of the geometry column of the input layer to use
+        for dissolving.
+   *  - **Produce one feature for each geometry in any kind of
+        geometry collection in the source file**
+      - ``EXPLODE_COLLECTIONS``
+      - [boolean]
+        
+        Default: False
+      - Produce one feature for each geometry in any kind of geometry
+        collection in the source file
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst]: I'm not sure I understand this one

File: source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst [hunk 5]
```diff
@@ -19,53 +19,224 @@ This algorithm is derived from the `ogr2ogr utility <https://gdal.org/ogr2ogr.ht
...
 
-``Additional creation Options`` [string]
-  Optional
+   *  - Label
+      - Name
+      - Type
+      - Description
+   *  - **Output layer**
+      - ``OUTPUT``
+      - [vector:any]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst]:
```suggestion
      - [vector: any]
```

File: source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst [hunk 6]
```diff
@@ -19,53 +19,224 @@ This algorithm is derived from the `ogr2ogr utility <https://gdal.org/ogr2ogr.ht
...
+   :widths: 20 20 20 40
+   :stub-columns: 0
 
+   *  - Label
+      - Name
+      - Type
+      - Description
+   *  - **Output layer**
+      - ``OUTPUT``
+      - [vector:any]
```
Comment (DelazJ) [inline source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst]:
```suggestion
      - [vector: any]
```

File: source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst (hunk 4 above)
Comment (havatv) [reply source/docs/user_manual/processing_algs/gdal/vectorgeoprocessing.rst]: Let's hope that the users will understand (it is cut and paste from the code, I think).

**Comment by havatv** [comment]
@DelazJ, I finally found the "Add suggestion to batch" button! :-)

**Comment by DelazJ** [comment]
> @DelazJ, I finally found the "Add suggestion to batch" button! :-)

😄 👍 I wonder why it's not in the main tab; it should be the default

**Pull request merged**